src/benchmark/RemotePingPong/Program.cs

```diff
@@ -188,6 +188,8 @@ private static async Task<(bool, long, int)> Benchmark(int numberOfClients, long
                 PrintSysInfo();
             }
 
+            var startThreads = Process.GetCurrentProcess().Threads.Count;
```

Comment:
Recorded after Akka.Remote association is established, but before pressure is applied.